js/main.js

```diff
@@ -28,22 +33,34 @@ var $parkAddressLine2 = document.querySelector('#park-address-line2');
 var $parkPhoneNumber = document.querySelector('#park-phone-number');
 var $parkEmail = document.querySelector('#park-email');
 
-var parkCode = null;
+var park = {
+  fullName: null,
+  photoUrl: null,
+  photoAlt: null,
+  parkCode: null
+};
+
+var newPark = {
+  fullName: null,
+  photoUrl: null,
+  photoAlt: null,
+  parkCode: null
+};
```

Comment:
These two variables seem **_very_** similar. What is their difference? Do you really need both? Maybe if you gave them more descriptive names, their purpose would seem more clear.